docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -95,7 +95,7 @@ This file contains several service definitions:
 - ``postgres`` - The database.
 - ``redis`` - `The redis <https://redis.io/>`__ - broker that forwards messages from scheduler to worker.
 
-Optionally, you can enable flower by adding ``--profile flower`` option, e.g. ``docker compose --profile flower up``, or by explicitly specifying it on the command line e.g. ``docker compose up flower``.
+Optionally, you can enable flower by adding ``--profile flower`` option, e.g. ``docker-compose --profile flower up``, or by explicitly specifying it on the command line e.g. ``docker-compose up flower``.
```

Comment:
`docker-compose` is old command
Please see https://github.com/apache/airflow/pull/28747#issuecomment-1372447406

Reply:
> `docker-compose` is obsolete

We are currently using Docker Compose. Thank you for your correction